awx/main/tests/functional/models/test_inventory.py

```diff
@@ -5,8 +5,8 @@
 
 # AWX
 from awx.main.models import Host, Inventory, InventorySource, InventoryUpdate, CredentialType, Credential, Job
-from awx.main.constants import CLOUD_PROVIDERS
 from awx.main.utils.filters import SmartFilter
+from awx.main.utils.plugins import CLOUD_PROVIDERS
```

Comment:
This one seems easy and doesn't require that subtest dance..
```suggestion
from awx.main.utils.plugins import discover_available_cloud_provider_plugin_names
```